README.md

```diff
@@ -189,7 +185,9 @@ relation with the Scheme Owner, and can check at the Scheme Owner whether other
 Access and usage control guarantees enforcement of data access and usage policies defined as part of the terms and conditions established 
 when data resources or services are published or negotiated between providers and consumers. An API proxy plays the role of the 
 Policy Enforcement Point (PEP) and requires an additional Policy Decision Point (PDP). 
-In the current release of i4Trust, [API Umbrella](https://github.com/FIWARE/api-umbrella) implements both PEP and PDP functionalities.
+In the current release of i4Trust, PEP and PDP functionalitie provided via libraries in [lua](https://github.com/FIWARE/lua-fiware-lib). The libraries can be used to implement plugins for various api-gateways, with a solution for the [Kong API-Gateway](https://github.com/Kong/kong) already [implemented by FIWARE](https://github.com/FIWARE/kong-plugins-fiware).
+The PDP functionality for authorization via VerifiableCredentials is provided as an additional component, the [DSBA-PDP](https://github.com/FIWARE/dsba-pdp) and can be integrated into the flow, using the same [Kong-Plugins](https://github.com/FIWARE/kong-plugins-fiware) as PEP.
+The [DSBA-PDP](https://github.com/FIWARE/dsba-pdp) uses the [EBSI Trusted Issuers Registry API](https://api-pilot.ebsi.eu/docs/apis/trusted-issuers-registry/latest#/) for evaluating if an issuer is allowed to issue certain credentials. The PDP provides a compatibility mechanism to use the [iShare delegation endpoint](https://dev.ishareworks.org/delegation/endpoint.html) as trusted issuers source. Additionally, the [iShare delegation endpoint](https://dev.ishareworks.org/delegation/endpoint.html) is used to evaluate the actual policies referenced by the roles in the received credential.
```

Comment:
Not sure what you mean with this statement "The PDP provides a compatibility mechanism to use the [iShare delegation endpoint](https://dev.ishareworks.org/delegation/endpoint.html) as trusted issuers source."
Also, please update the iSHARE URLs to iSHARE.eu instead of ishareworks.org

Reply:
- the PDP can also use other Trusted Issuers Lists, for like they are defined by EBSI. To be used for i4Trust, it provides a layer to use the AR as a trusted issuers registry, the same way it would use others
- where can I find the api-documentation on ishare.eu? I was (until now) only aware of the ishareworks.org doc.

Reply:
ok, well, its same link, with different domain. ishareworks.org is old domain which is replaced with iSHARE.eu, we cannot guarantee for how long we will maintain ishareworks.org domain. so dev portal is now dev.iSHARE.eu but refers to same site